### PR TITLE
Update missing paths from go/wbb-oc-paths to ISIS and QOS Feature pro…

### PR DESCRIPTION
**Pull request by hariramnat**
…files

### CODE REVIEW

File: feature/isis/feature.textproto [hunk 1]
```diff
@@ -146,6 +146,26 @@ config_path {
...
+}
+telemetry_path {
+  path: "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/ipv6-reachability/prefixes/prefix/state/x-bit"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/ipv6-reachability/prefixes/prefix/subtlvs/subtlv/flags/state/flags"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/protocols/protocol/isis/levels/level/system-level-counters/state/part-changes"
+}
```
Comment (dplore) [inline feature/isis/feature.textproto:168]: maybe move to a new commented stanza called system level

File: feature/qos/feature.textproto [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+}
+config_path {
+  path: "/qos/interfaces/interface/output/queues/queue/config/name"
+}
+config_path {
+  path: "/qos/interfaces/interface/output/scheduler-policy/config/name"
+}
+telemetry_path {
+  path: "/qos/queues/queue/state/name"
+}
```
Comment (dplore) [inline feature/qos/feature.textproto:58]: perhaps add a commented stanza called 'queues'

**Review by dplore — CHANGES_REQUESTED** [review]
A few minor nits

File: feature/isis/feature.textproto (hunk 1 above)
Comment (hariramnat) [reply feature/isis/feature.textproto:170]: Done!

File: feature/qos/feature.textproto (hunk 2 above)
Comment (hariramnat) [reply feature/qos/feature.textproto:60]: Done!

**Review by hariramnat — COMMENTED** [review]
Added above changes

**Review by dplore — APPROVED** [review]

**Pull request merged**